lib/bitstyles_phoenix/components/form.ex

```diff
@@ -109,7 +109,7 @@ defmodule BitstylesPhoenix.Components.Form do
   end
 
   @doc ~S"""
-  Renders `<select>` elements, with the associated `<label>`s, and any errors for that field. As with the `select` form helper in `Phoenix.HTML.Form`, this expects the form, field, and options as first parameters.
+  Renders `<select>` elements, with the associated `<label>`s, and any errors for that field. As with the `select` form helper in `PhxForm`, this expects the form, field, and options as first parameters.
```

Comment:
this could probably stay `Phoenix.HTML.Form`, right?

Reply:
my bad, search&regex fu